README.md

```diff
@@ -198,7 +198,7 @@ A very common thing is to add [`:autoMergePatch`](https://docs.renovatebot.com/p
 ### python
 
 **There is currently only [limited / "alpha level" support](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/pip-compile/readme.md) for python using `pip-compile`.
-This configuration has not been used with python projects for a while, it most certainly needs changes.**
+This configuration has been used in some python projects for a while, you can test it in [bettermarks/renovate-config-test-pip-compile](https://github.com/bettermarks/renovate-config-test-pip-compile) by replacing the dependency files and change the config and see what updates renovate files.**
```

Comment:
```suggestion
This configuration is currently only used in a small number of python projects, so it may need modifications. You can test it in [bettermarks/renovate-config-test-pip-compile](https://github.com/bettermarks/renovate-config-test-pip-compile) by modifying the dependency files and the renovate config in order to see what updates renovate produces.**
```